frontend/public/components/utils/datetime.ts

```diff
@@ -82,6 +82,51 @@ export const fromNow = (dateTime, now=undefined) => {
   return `${-Math.round(daysAgo / 365)} years from now`;
 };
 
+export const fromNowSimple = (dateTime, now=undefined) => {
```

Comment:
I'm hesitant to want to duplicate all that code && just update the text strings.  I wonder if there is a better way to do this. No suggestion atm, but thinking on it.

Reply:
Suggest a `simple = false` parameter instead of duplicating all the logic.

We need to add tests to the existing datetime tests for this

Reply:
Actually I'd call it `noSuffix = false` or something like that. We should try to stay consistent with moment.js in case we ever add the library back. (I'm not sure removing it was really a good thing.)

See https://momentjs.com/docs/#/displaying/fromnow/